applications/swimming_DEM_application/custom_utilities/fields/pouliot_flow_field_2D.cpp

```diff
@@ -15,16 +15,16 @@ void PouliotFlowField2D::ResizeVectorsForParallelism(const int n_threads)
 void PouliotFlowField2D::UpdateCoordinates(const double time, const array_1d<double, 3>& coor, const int i_thread)
 {
     if (!mCoordinatesAreUpToDate[i_thread]){
-        mExpX[i_thread] = std::exp(- 25 * coor[0]);
-        mExpY[i_thread] = std::exp(- 25 * coor[1]);
+        mX[i_thread] = coor[0];
+        mY[i_thread] = coor[1];
     }
 }
 
 void PouliotFlowField2D::UpdateCoordinates(const double time, const vector<double>& coor, const int i_thread)
```

Comment:
Same...

Reply:
I use the style mCapitalLetters for member veriables everywhere. Same answer as above.